wc-calypso-bridge.php

```diff
@@ -34,6 +34,24 @@
 define( 'WC_CALYPSO_BRIDGE_CURRENT_VERSION', '1.8.0' );
 define( 'WC_MIN_VERSION', '3.0.0' );
 
+if ( ! function_exists( 'wc_calypso_bridge_is_personal_plan' ) ) {
+	/**
+	 * Returns if a site is a personal plan site (i.e. WOOP).
+	 * The `at_options` array is created during provisioning. Usually it is 'business' or 'ecommerce'
+	 * To Test: update_option( 'at_options', array( 'plan_slug' => 'personal' ) );
+	 * or: wp option set at_options --format=json '{ "plan_slug": "personal" }'
+	 *
+	 * @return bool True if the site is a personal site.
+	 */
+	function wc_calypso_bridge_is_personal_plan() {
+		if ( class_exists( 'Atomic_Plan_Manager' ) && method_exists( 'Atomic_Plan_Manager', 'current_plan_slug' ) ) {
+			return Atomic_Plan_Manager::current_plan_slug() === Atomic_Plan_Manager::PERSONAL_PLAN_SLUG;
```

Comment:
Having to gate access to this class isn't ideal. I'm tempted to say just hard code the slugs anywhere outside of wpcomsh. And in turn use a single wrapper function to get the slug in bridge that can be filtered before use, we then centralize the gating to one function. We're creating plan changes overhead with these named functions regardless so we may as well just skip the complexity and hard code the slugs directly.

Alternatively we could extract the dependency (or a stub) out with composer.